coffee/directives/fixtable.coffee

```diff
@@ -76,15 +84,29 @@ angular.module 'fixtable'
 					scope.loading = newValue
 
 			# get new page data
-			getPageData = ->
+			getPageData = (reload = false) ->
 				cb = scope.$parent[scope.options.pagingOptions.callback]
-				cb scope.options.pagingOptions, scope.options.sort, scope.appliedFilters
+				cb scope.options.pagingOptions, scope.options.sort, scope.appliedFilters, reload
 
 			# provide methods to page forward/back in footer template
-			scope.nextPage = ->
-				scope.pagingOptions.currentPage += 1
-			scope.prevPage = ->
-				scope.pagingOptions.currentPage -= 1
+			do setPagingActions = ->
+				if scope.options.pagingOptions?.type is 'prevNext'
```

Comment:
- [ ] Extract this mode into a const - probably add a comment that explains the default mode and possible modes